mozilla-release/browser/components/preferences/in-content/privacy.xul

```diff
@@ -861,6 +871,7 @@
     I KID YOU NOT.
   -->
   <radiogroup id="certSelection"
```

Comment:
@tsl143 could you please make sure that there is only 1 element with id="certSelection" (see line 854/864)?

Reply:
Lets keep it for different ticket scope, some of the evils we inherited from Firefox

Reply:
I do not mind. But could you provide any TODO comments out there so we would not forget in the future to fix it?